include/proto.hpp

```diff
@@ -81,6 +81,7 @@ CatRef shopStorageRoom(const UniqueRoom *shop);
 
 // config.cpp
 int getPkillInCombatDisabled();
+int getPlayersCanMudmailStaff();
```

Comment:
Put this inside of Config instead.

Reply:
This needs to get put inside of the Config class in config.hpp instead of proto.h  It should also match the signature you used in Config.cpp (bool vs int)